### PR TITLE
Release Highlights 2.5

### PR DESCRIPTION
**Pull request by gdziuba**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by gdziuba** [comment]
We need to confirm that all features ship tomorrow before we merge.

File: src/blog/2024/06/flowfuse-2-5-release.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: "FlowFuse 2.5: New features to visualize snapshots, LDAP integration, and more"
+subtitle: Enhancing security, visualization, and deployment flexibility.
+description: Discover the new features in FlowFuse 2.5, including LDAP integration, visual snapshot comparisons, blueprint previews, snapshot import/export support, and custom domain deployment for dashboards and APIs.
+date: 2024-06-07
+authors: ["grey-dziuba"]
+image: /blog/2024/06/images/release-2-5-graphic.png
+tags:
+   - posts
+   - flowfuse
+   - releases
```
Comment (Hasmin-AC) [inline src/blog/2024/06/flowfuse-2-5-release.md:11]:
```suggestion
   - releases
   - LDAP
   - snapshot
   - blueprints
   - Node-RED
   
```

File: src/blog/2024/06/flowfuse-2-5-release.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+In our commitment to making FlowFuse more versatile and secure, we've introduced support for LDAP. This has been a much requested feature that we are excited to release.  This allows organizations to manage user authentication and authorization seamlessly, leveraging their existing LDAP infrastructure. With this feature, you can ensure that access controls are robust and aligned with your company's security policies.
+
+## Compare Snapshots Visually to see differences
+
+Keeping track of changes and updates has never been easier. With our new visual comparison tool, you can now compare snapshots and see the differences at a glance. This feature provides a clear, graphical representation of changes, making it simple to identify modifications and understand their impact.
+
+## Preview Blueprints before Deployment
+
+Similarly to the feature to be able to compare snapshots we have enabled this same feature to allow users to preview team flows and blueprints prior to deploying.  This expedites the process when you are attempting to find the correct flow for your application.  We find that our users will be able to
```
Comment (Hasmin-AC) [inline src/blog/2024/06/flowfuse-2-5-release.md:29]: missing content at the end

File: src/blog/2024/06/flowfuse-2-5-release.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Keeping track of changes and updates has never been easier. With our new visual comparison tool, you can now compare snapshots and see the differences at a glance. This feature provides a clear, graphical representation of changes, making it simple to identify modifications and understand their impact.
+
+## Preview Blueprints before Deployment
+
+Similarly to the feature to be able to compare snapshots we have enabled this same feature to allow users to preview team flows and blueprints prior to deploying.  This expedites the process when you are attempting to find the correct flow for your application.  We find that our users will be able to 
+
+## Import and Export support for snapshots
+
+Managing your Node-RED artifacts just got simpler with our new import and export support for snapshots. You can now easily transfer configurations between deployment or back up your current setup with just a few clicks. This feature is perfect for those who need to replicate environments within or ensure their configurations are safely stored.
```
Comment (Hasmin-AC) [inline src/blog/2024/06/flowfuse-2-5-release.md:33]: within what?

File: src/blog/2024/06/flowfuse-2-5-release.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+authors: ["grey-dziuba"]
+image: /blog/2024/06/images/release-2-5-graphic.png
+tags:
+   - posts
+   - flowfuse
+   - releases
+
+---
+
+FlowFuse 2.5 introduces LDAP integration, Snapshot comparison, Extends the ability to preview flow to Blueprints, rounds out the management for snapshots, while lastly enabling customer to deploy FlowFuse with your own domain name.
```
Comment (knolleary) [inline src/blog/2024/06/flowfuse-2-5-release.md:15]:
```suggestion
FlowFuse 2.5 introduces LDAP integration, snapshot comparison, extends the ability to preview flow to Blueprints, rounds out the management for snapshots, and allows you to point your own domain names at your FlowFuse instances.
```

File: src/blog/2024/06/flowfuse-2-5-release.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+<!--more-->
+
+## Support for LDAP integration
+
+In our commitment to making FlowFuse more versatile and secure, we've introduced support for LDAP. This has been a much requested feature that we are excited to release.  This allows organizations to manage user authentication and authorization seamlessly, leveraging their existing LDAP infrastructure. With this feature, you can ensure that access controls are robust and aligned with your company's security policies.
```
Comment (knolleary) [inline src/blog/2024/06/flowfuse-2-5-release.md:21]:
```suggestion
In our commitment to making FlowFuse more versatile and secure, we've introduced support for LDAP alongside our existing SAML SSO options. This has been a much requested feature that we are excited to release.  This allows organizations to manage user authentication seamlessly, leveraging their existing LDAP infrastructure. With this feature, you can ensure that access controls are robust and aligned with your company's security policies.
```

Technically, we only support authentication.

**Review by knolleary — DISMISSED** [review]
Some suggested changes.

Also, would be good to still have the other standard sections we have in release posts - link to the full list of changes, what's next, upgrading, trying it out CTA etc.

File: src/blog/2024/06/flowfuse-2-5-release.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+In our commitment to making FlowFuse more versatile and secure, we've introduced support for LDAP alongside our existing SAML SSO options. This has been a much requested feature that we are excited to release.  This allows organizations to manage user authentication seamlessly, leveraging their existing LDAP infrastructure. With this feature, you can ensure that access controls are robust and aligned with your company's security policies.
+
+## Compare Snapshots Visually to see differences
+
+Keeping track of changes and updates has never been easier. With our new visual comparison tool, you can now compare snapshots and see the differences at a glance. This feature provides a clear, graphical representation of changes, making it simple to identify modifications and understand their impact.
+
+## Preview Blueprints before Deployment
+
+Similarly to the feature to be able to compare snapshots we have enabled this same feature to allow users to preview team flows and blueprints prior to deploying.  This expedites the process when you are attempting to find the correct flow for your application.  We find that our users will be able to
```
Comment (gdziuba) [inline src/blog/2024/06/flowfuse-2-5-release.md:34]:
```suggestion
Similarly to the feature to be able to compare snapshots we have enabled this same feature to allow users to preview team flows and blueprints prior to deploying.  This expedites the process when you are attempting to find the correct flow for your application.  We find that our users will be able to expedite the exploration of both their own team library and the blueprints.
```

File: src/blog/2024/06/flowfuse-2-5-release.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+Keeping track of changes and updates has never been easier. With our new visual comparison tool, you can now compare snapshots and see the differences at a glance. This feature provides a clear, graphical representation of changes, making it simple to identify modifications and understand their impact.
+
+## Preview Blueprints before Deployment
+
+Similarly to the feature to be able to compare snapshots we have enabled this same feature to allow users to preview team flows and blueprints prior to deploying.  This expedites the process when you are attempting to find the correct flow for your application.  We find that our users will be able to expedite the exploration of both their own team library and the blueprints.
+
+## Import and Export support for snapshots
+
+Managing your Node-RED artifacts just got simpler with our new import and export support for snapshots. You can now easily transfer configurations between deployment or back up your current setup with just a few clicks. This feature is perfect for those who need to replicate environments within or ensure their configurations are safely stored.
```
Comment (gdziuba) [inline src/blog/2024/06/flowfuse-2-5-release.md:38]:
```suggestion
Managing your Node-RED artifacts just got simpler with our new import and export support for snapshots. With just a few clicks, you can easily transfer configurations between deployments or back up your current setup. This feature is perfect for those who need to replicate environments within their own enterprise with multiple deployments or ensure their configurations are safely stored.
```

Comment (gdziuba) [reply src/blog/2024/06/flowfuse-2-5-release.md:38]: Feels redundant now.

**Comment by gdziuba** [comment]
> Some suggested changes.
> 
> Also, would be good to still have the other standard sections we have in release posts - link to the full list of changes, what's next, upgrading, trying it out CTA etc.

Thanks.  For pointing out.  I guess I was focused on the core documentation I skipped that part.  adding.

**Review by Hasmin-AC — APPROVED** [review]

**Pull request merged**